plugins/huggingface/plugin.go

```diff
@@ -0,0 +1,22 @@
+package huggingface
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "huggingface",
+		Platform: schema.PlatformInfo{
+			Name:     "HuggingFace",
```

Comment:
```suggestion
			Name:     "Hugging Face",
```

Reply:
The brand name has a space, so could we audit that we are maintaining that throughout the code?

Reply:
thanks arun, i have addressed the change